Packages/com.unity.inputsystem/InputSystem/Devices/Gamepad.cs

```diff
@@ -86,7 +86,7 @@ public struct GamepadState : IInputStateTypeInfo
         internal const string ButtonSouthShortDisplayName = "Cross";
         internal const string ButtonNorthShortDisplayName = "Triangle";
         internal const string ButtonWestShortDisplayName = "Square";
-        internal const string ButtonEastShortDisplayName = "East";
+        internal const string ButtonEastShortDisplayName = "Circle";
```

Comment:
Good catch, but realize this is solved in a very platform-dependent way in what is supposed to be platform-agnostic code. Will feed this into a doc I am creating on the subject.